third_party/pkg/split/lib/split.dart

```diff
@@ -65,7 +65,7 @@ class Splitter {
 
   external List getSizes();
 
-  external void collapse();
+  external void collapse(int indexToCollapse);
```

Comment:
I'm wondering on the context of this change. It appears unconnected to the rest of the PR, and I'm wondering why it hasn't caused other required changes?

Reply:
A working version of the collapse() method is needed so we can maximize the UI output panel (by collapsing the code panel).  
   The underlying JS library defined collapse() as taking the index of the panel to collapse to it's minimum size, and the dart wrapper incorrectly defined no parameters (as you can see).

  I use the (now working) version of this method to call collapse(0) to collapse the code panel and thereby maximize the UI panel.   (Otherwise all we can do is being up the UI panel in at it's minimum size (100px), which is all the split library will do without it's collapse() method.  

   The collapse() method was never used anywhere else in the project (as it would not have worked anyway, because undefined was being passed to the JS collapse() method which was expecting an index).